rlp/rlpgen/gen.go

```diff
@@ -635,6 +678,9 @@ func (bctx *buildContext) makeOp(name *types.Named, typ types.Type, tags rlpstru
 		if isBigInt(typ) {
 			return bigIntOp{}, nil
 		}
+		if isUint256(typ) {
+			return uint256Op{pointer: true}, nil
```

Comment:
Why is this pointer true?

Reply:
Thanks, I've missed it! I'll fix

https://github.com/ethereum/go-ethereum/pull/26898/files#diff-1bfd65d008fe8358562b840df33d80ef9210d68f9f295fc9558d644a45f6396cR682

Reply:
By introduce [this fix](https://github.com/kaiachain/kaia/pull/233/commits/4680c5eb46b3ad0fe256a999e4bc5bce83729531), it seems we need to fix [uint256.out.txt](https://github.com/kaiachain/kaia/pull/233/files#diff-782511bf28abf012ba50c201bef716f02aaa5a2558633755f9f2743382565ab4).